cmd/qat_plugin/README.md

```diff
@@ -3,26 +3,18 @@
 Table of Contents
 
 * [Introduction](#introduction)
-    * [Modes and Configuration options](#modes-and-configuration-options)
+* [Modes and Configuration options](#modes-and-configuration-options)
 * [Installation](#installation)
     * [Prerequisites](#prerequisites)
-    * [Pre-built image](#pre-built-image)
-    * [Getting the source code:](#getting-the-source-code)
-    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
-        * [Build the plugin image](#build-the-plugin-image)
-        * [Deploy the DaemonSet](#deploy-the-daemonset)
-        * [Verify QAT device plugin is registered:](#verify-qat-device-plugin-is-registered)
-    * [Deploying by hand](#deploying-by-hand)
-        * [Build QAT device plugin](#build-qat-device-plugin)
-        * [Deploy QAT plugin](#deploy-qat-plugin)
-    * [QAT device plugin Demos](#qat-device-plugin-demos)
-        * [DPDK QAT demos](#dpdk-qat-demos)
-            * [DPDK Prerequisites](#dpdk-prerequisites)
-            * [Build the image](#build-the-image)
-            * [Deploy the pod](#deploy-the-pod)
-            * [Manual test run](#manual-test-run)
-            * [Automated test run](#automated-test-run)
-        * [OpenSSL QAT demo](#openssl-qat-demo)
+    * [Pre-built images](#pre-built-images)
+* [Demos and Testing](#demos-and-testing)
+    * [DPDK QAT demos](#dpdk-qat-demos)
+        * [DPDK Prerequisites](#dpdk-prerequisites)
+        * [Build the image](#build-the-image)
+        * [Deploy the pod](#deploy-the-pod)
+        * [Manual test run](#manual-test-run)
+        * [Automated test run](#automated-test-run)
```

Comment:
Maybe later build instructions here should go with the demo itself, with this having just link to that?

Reply:
you mean just keep `#dpdk-qat-demos` anchor?

Reply:
On closer look, the document itself is lacking "Build the image" section listed in TOC, so that TOC item should be just removed.

Btw. later in the doc,  `#### DPDK QAT demos` should be `### DPDK QAT demos` to make heading match TOC structure.

Reply:
thanks! I fixed these. I dropped "Build the image" section but forgot to update the TOC list. It's fixed now.